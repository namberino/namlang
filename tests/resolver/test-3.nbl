fun test()
{
    var t = "1";
    var t = "2"; // error
}
