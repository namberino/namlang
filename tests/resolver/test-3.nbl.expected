Executing file: tests/resolver/test-3.nbl
On line: 4, Error at 't': Already a variable with this name in this scope
